input/docs/getting-started/running-cake-scripts.md

```diff
@@ -0,0 +1,45 @@
+Order: 10
+---
+
+There are different runners available for running Cake scripts.
+
+# .NET Core Tool
+
+:::{.alert .alert-info}
+This is the recommended way to run Cake Scripts.
+:::
+
+The [Cake.Tool](https://www.nuget.org/packages/Cake.Tool) NuGet package, is a .NET Core global tool compiled for .NET Core 2.1 or newer.
+
+It allows you to easily install Cake and have it globally available in your environment using the dotnet tool command:
+
+```shell
+dotnet tool install -g Cake.Tool --version x.y.z
```

Comment:
Is global install the recommended approach? Not local manifest tool?

Reply:
Yes I think tool manifest should be the recommended way.

Reply:
Updated documentation to local tool using a tool manifest, including a note that it requires .NET Core 3.0 with a link to Microsoft documentation showing other ways to install tools.